translations/fr/try_ruby_130.md

```diff
@@ -0,0 +1,13 @@
+---
+lang:   FR
+title:  Un d'entre eux lève la main
```

Comment:
I would suggest `L'un d'entre eux` rather than `Un d'entre eux`, which sounds better to me.